src/Error/ErrorHandler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * ErrorHandler class
+ *
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         0.10.5
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace DebugKit\Error;
+
+use Cake\Core\App;
+use Cake\Core\Plugin;
+use Cake\Error\ErrorHandler as BaseErrorHandler;
+use Cake\Http\ResponseEmitter;
+use DebugKit\ToolbarService;
+use Exception;
+use Throwable;
+
+/**
+ * @see \Cake\Error\ExceptionRenderer for more information
+ */
+class ErrorHandler extends BaseErrorHandler
```

Comment:
Should the plugin register this error handler in its bootstrap hook? That would make the deprecation workflow with debugkit enabled easier.

Reply:
>Should the plugin register this error handler in its bootstrap hook?
just need to register in `app/config/bootstrap.php`

Reply:
how we can enable it in `{plugin}config/bootstrap.php` ?

Reply:
Instead of an ErrorHandler class we could do something like the following in debug kit's bootstrap:

```php
$previousHandler = set_error_handler(
    function ($code, $message, $file, $line) use (&$previousHandler) {
        if ($code == E_USER_DEPRECATED || $code == E_DEPRECATED) {
            DeprecatedPanel::addDeprecation($code, $message, $file, $line);
            return;
        }
        if ($previousHandler) {
            return $previousHandler($code, $message, $file, $line);
        }
    }
);
```

This approach would remove the deprecations from the logs and HTML display, and only display them in DebugKit.